src/libcpu/src/jit/jit_internal.h

```diff
@@ -108,6 +110,7 @@ class PPCEmuAssembler : public asmjit::X86Assembler
       PPCMemRef(lrMem, lr);
       PPCMemRef(ctrMem, ctr);
 
+      PPCMemRef(ciaMem, cia);
```

Comment:
Can't do that since it's needed at runtime, not translation time. I can pass it as an extra argument to the verify functions, but it seems to me that your description of cia as a debugging tool exactly matches the purpose to which it's being put here.